### ISSUE TITLE
Summarizing the need

### ISSUE THREAD

**Issue by aaronshaf** [WebReflection/html-parsed-element#2]
Can you summarize the need for this? There is a lot in the [linked conversation](https://github.com/w3c/webcomponents/issues/551).

**Comment by franktopel** [comment]
May I ask for what purpose are you asking a summary?

**Comment by aaronshaf** [comment]
If I include this lib into my own base class for custom elements, I'd like to be able to explain and understand the reason.

Thanks for all you do.

**Comment by franktopel** [comment]
As this served as a base for development of this package, does this gist help you out already? https://gist.github.com/franktopel/5d760330a936e32644660774ccba58a7

**Comment by aaronshaf** [comment]
Yes, thanks.
